issueWork/UI/UX__issue/README.md

```diff
@@ -0,0 +1,27 @@
+# UI / UX issue  
+
+## Changes made:  
+
+> 1. Complete overhall of webpage layout based on previously received feedback. Essentially creating a clean and somewhat minimal style.
+ - ### Laws:  
+   - **Aesthetic-Usability**
+   - **Jakob's Law**
+   - **Hick's Law**
+> 2. Attached and created an accordin to the hero section's breif about me. Which when clicked expands to a larger more indepth about me section.  
```

Comment:
Use [hemingway](https://hemingwayapp.com/) or Grammarly when writing copy. You have a lot of spelling errors.

Reply:
No worries, documentation is a huge skill that folks are hiring for. So working on communication is going to be key. It looks a lot better after your edits. Thanks.